CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# Catch-all
+* @tgraf @manalibhutiyani
+
+# BPF Data Path
+bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
```

Comment:
Let's use `bpf/` here and in all other places consistently. This ensures that subdirectories get matched as well. `bpf/*` will only match one nesting level.